nntrainer/layers/acti_func.cpp

```diff
@@ -221,6 +217,7 @@ Tensor &ActiFunc::softmaxPrime(Tensor const &x, Tensor &output,
   const float *d = derivative.getData();
   float *pp = output.getData();
 
+  Tensor tmp = Tensor(1, 1, 1, width);
```

Comment:
why do we need this?

Reply:
By using temporal tensor we could support in-place operation. 
Further more in the forwarding process of softmax we've already used temporal tensor exactly same size with this tensor. (please refer https://github.com/nnstreamer/nntrainer/blob/main/nntrainer/layers/acti_func.cpp#L191)